OracleHTTPServer/dockerfiles/12.2.1.4.0/container-scripts/stop-ohs.py

```diff
@@ -9,8 +9,8 @@
 java_home = os.environ.get("JAVA_HOME", "/usr/java/default")
 ohs_comp_name   = os.environ.get("OHS_COMPONENT_NAME", "ohs1")
 domain_name  = os.environ.get("DOMAIN_NAME", "ohsDomain")
-domain_path  = '/u01/oracle/ohssa/user_projects/domains/' + domain_name
-oracle_home = os.environ.get("ORACLE_HOME", "/u01/oracle/ohssa")
+domain_path  = os.environ.get("DOMAIN_HOME", "temp")
```

Comment:
You could use the default path instead of temp.

Reply:
defaulted domain home